charts/backstage/ci/serviceMonitorRelabelings-values.yaml

```diff
@@ -0,0 +1,13 @@
+metrics:
+  serviceMonitor:
```

Comment:
This test case has no effect but offers a false impression. `ServiceMonitor` is not rendered unless the` metrics.enabled` value is set to true. I think we either have to adjust/come up with a way to test this properly with `metrics.enabled` (`ServiceMonitor` CRD needs to be present in CI) or we should remove this test case.